app/models/concerns/audit_associations.rb

```diff
@@ -50,13 +50,22 @@ module ClassMethods
 
       def audited(options = {})
         if options[:associations].present?
-          include DirtyAssociations
-
-          dirty_has_many_associations(*(Array(options[:associations])))
+          configure_dirty_associations(Array(options[:associations]))
         end
 
         super
       end
+
+      def audit_associations(associations)
+        new_associations = Array(associations)
+        configure_dirty_associations(new_associations)
+        self.audited_options[:associations] = Array(self.audited_options[:associations]) + new_associations
```

Comment:
I think this should be `|` instead of `+` in case two plugins try to add the same associations.

Reply:
good point, added `|`